tests/python/contrib/test_hexagon/test_vtcm_bandwidth.py

```diff
@@ -123,15 +125,18 @@ class TestMatMulVec:
 
     # Removed most of these to speedup CI.
     size = tvm.testing.parameter(
-        # 10 * KB,
+        128,
+        256,
+        1024,
+        10 * KB,
         # 20 * KB,
         # 40 * KB,
         # 80 * KB,
         # 160 * KB,
         # 320 * KB,
         640 * KB,
         # MB,
-        # 2 * MB,
```

Comment:
Did you mean to uncomment this?  Makes the test run longer in CI.

Reply:
Added a check if running in CI.